### PR TITLE
Cleanup deprecation warnings, disambiguate new static settings

### PR DESCRIPTION
**Pull request by bigtimebuddy**
### Chore

* Remove `sharedLoader` from the Application options documentation
* Remove `PIXI.` from deprecation warnings
* Renamed `BatchRenderer.maxTextures` to `BatchRenderer.defaultMaxTextures`
* Renamed `BatchRenderer.batchSize` to `BatchRenderer.defaultBatchSize`
* Renamed `Filter.resolution` to `Filter.defaultResolution`
* Renamed `Filter.multisample` to `Filter.defaultMultisample`
* Decentralize settings types using GlobalMixins

### CODE REVIEW

File: packages/core/src/batch/BatchRenderer.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ export class BatchRenderer extends ObjectRenderer
     get MAX_TEXTURES(): number
     {
         // #if _DEBUG
-        deprecation('7.1.0', 'PIXI.BatchRenderer.MAX_TEXTURES renamed to PIXI.BatchRenderer.maxTextures');
+        deprecation('7.1.0', 'BatchRenderer#MAX_TEXTURES renamed to BatchRenderer#maxTextures');
```
Comment (Zyie) [inline packages/core/src/batch/BatchRenderer.ts:298]: This should be `renamed to BatchRenderer.defaultMaxTextures`?
Also there are '#' instead of '.'

Comment (Zyie) [reply packages/core/src/batch/BatchRenderer.ts:298]: ahh! 

got used to seeing static properties in these PR's 😄

**Review by Zyie — APPROVED** [review]

File: packages/core/src/batch/BatchRenderer.ts (hunk 1 above)
Comment (bigtimebuddy) [reply packages/core/src/batch/BatchRenderer.ts:298]: Threw in this change for free 😝

Comment (bigtimebuddy) [reply packages/core/src/batch/BatchRenderer.ts:298]: TITLE_CASING for members is not great

**Pull request merged**